libs/vkd3d/resource.c

```diff
@@ -1009,7 +1009,10 @@ HRESULT vkd3d_create_buffer(struct d3d12_device *device,
     if (heap_type == D3D12_HEAP_TYPE_UPLOAD)
         buffer_info.usage &= ~VK_BUFFER_USAGE_TRANSFER_DST_BIT;
     else if (heap_type == D3D12_HEAP_TYPE_READBACK)
-        buffer_info.usage = VK_BUFFER_USAGE_TRANSFER_DST_BIT;
+    {
+        buffer_info.usage = VK_BUFFER_USAGE_TRANSFER_DST_BIT
+                | VK_BUFFER_USAGE_STORAGE_BUFFER_BIT;
```

Comment:
This formatting looks goofy, should probably just be on one line.

Reply:
Should mark this with a comment saying why it's used so we can remember to remove it in the future if we implement scratch buffer resolve.

Reply:
Nvm, we'll need SSBO usage anyways ...